File: plugin/src/org/jetbrains/yesod/lucius/parser/LuciusCompositeElementType.java
package org.jetbrains.yesod.lucius.parser;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.yesod.lucius.LuciusLanguage;

/**
 * @author Leyla H
 */

public class LuciusCompositeElementType extends IElementType {

    String myDebugName;

    public LuciusCompositeElementType(@NotNull @NonNls String debugName) {
        super(debugName, LuciusLanguage.INSTANCE);
    }

    public String getDebugName() {
        return myDebugName;
    }
}

File: plugin/src/org/jetbrains/yesod/lucius/parser/LuciusCompositeElementType.kt
package org.jetbrains.yesod.lucius.parser

/**
 * @author Leyla H
 */

import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import org.jetbrains.yesod.lucius.LuciusLanguage


class LuciusCompositeElementType(val debugName: String) :
        IElementType(debugName, LuciusLanguage.INSTANCE)